.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+const { overrides } = require('@netlify/eslint-config-node')
+
+const customOverrides = [
+  {
+    files: ['*.js'],
+    rules: {
```

Comment:
Note: instead of using overrides, it it possible to put those `rules` directly under the top-level `rules` property.